src/Tools/Github/GitMergeBot/Program.cs

```diff
@@ -144,10 +150,24 @@ private Program(Options options)
             return branchName;
         }
 
+        private async Task CommentOnIssue(string user, string repo, int issueNumber, string message)
+        {
+                var resp = await _client.Connection.Post<string>(
```

Comment:
Wow, I didn't see that there.  This API is really confusing.

To get pull-request comments info, you write `_client.Repository.PullRequest.Comment`, but to get issue-comments (which is what we want in this case), it's `_client.Issue.Comment.Create`.  Really odd design choice.